lib/flight_cert/cli.rb

```diff
@@ -82,6 +82,15 @@ def self.create_command(name, args_str = '')
       c.slop.bool '--config-only', 'Only update the internal configuration, skips certificate generation'
     end
 
+    create_command 'cert-install' do |c|
+      c.summary = 'Use a self-generated SSL certificate'
+      c.description = <<~DESC
+        Use an SSL certificate that has been generated by the user, independent of Flight Cert.
```

Comment:
Depending on how Flight Cert has been installed, it may or may not be called `Flight Cert`.  The correct name for it can be found at `Flight.config.program_name`.  I'll explain more on this when we meet later today.